src/main/java/seedu/address/logic/parser/Parser.java

```diff
@@ -12,5 +12,5 @@
      * Parses {@code userInput} into a command and returns it.
      * @throws ParseException if {@code userInput} does not conform the expected format
      */
-    T parse(String userInput) throws ParseException;
+    T parse(String args) throws ParseException;
```

Comment:
I think userInput might've been a better name for this? easier to understand imo args q vague